.travis.yml

```diff
@@ -0,0 +1,54 @@
+language: node_js
+
+sudo: required
+
+matrix:
+  include:
+  - name: "tests"
+    node_js: "11"
```

Comment:
should we use v12 here too?

Reply:
nope because the test framework (ganache) currently is only v11. Will monitor it and then switch back to 12